setup.py

```diff
@@ -115,6 +115,7 @@ def run(self):
         'dataclasses-json',
         'pyyaml',
         'graphviz',
-        'pytact @ git+https://git@github.com/coq-tactician/coq-tactician-reinforce#f6992c129ad99aa47fb80a6322f6ead5e86bda92'
+        #'pytact @ git+https://git@github.com/coq-tactician/coq-tactician-reinforce#f6992c129ad99aa47fb80a6322f6ead5e86bda92'
```

Comment:
Can you delete this line?  I think it is a bad habit to comment out code like this.  We can always use git blame and the file history to figure out what this used to be.

Reply:
done